In [1]:
import numpy as np
import pandas as pd

In [2]:
df = pd.read_csv('USDA.csv')

In [3]:
df.head()

SortOrder           Region Region or State  Year  Total land  \
0          1  Northeast total       Northeast  1945      112402   
1          2  Northeast total       Northeast  1949      112378   
2          3  Northeast total       Northeast  1954      112378   
3          4  Northeast total       Northeast  1959      112324   
4          5  Northeast total       Northeast  1964      112223   

   Total cropland  Cropland used for crops  Cropland used for pasture  \
0           25027                    20904                       2280   
1           24557                    20340                       4217   
2           22830                    16635                       3777   
3           20973                    15189                       3217   
4           19173                    14016                       2559   

  Cropland idled  Grassland pasture and range  Forest-use land (all)  \
0           1842                        10126                  61788   
1           N.A.                         6928                  62759   
2           2418                         7186                  63537   
3           2567                         7999                  63845   
4           2598                         7110                  66637   

   Forest-use land grazed  Forest-use land not grazed  \
0                    9001                       52787   
1                    7097                       55662   
2                    5588                       57949   
3                    3488                       60357   
4                    3573                       63064   

   All special uses of land  Land in rural transportation facilities  \
0                      6374                                     1910   
1                      7797                                     1921   
2                      7497                                     2032   
3                      7703                                     2100   
4                      8210                                     2120   

   Land in rural parks and wildlife areas  \
0                                    2818   
1                                    4234   
2                                    4083   
3                                    4371   
4                                    4845   

   Land in defense and industrial areas  \
0                                   529   
1                                   524   
2                                   612   
3                                   599   
4                                   617   

   Farmsteads, roads, and miscellaneous farmland  Land in urban areas  \
0                                           1118                 3975   
1                                           1118                 4126   
2                                            770                 4137   
3                                            633                 5821   
4                                            628                 6186   

   Other land  
0        5112  
1        6211  
2        7191  
3        5983  
4        4907

In [4]:
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 780 entries, 0 to 779
Data columns (total 20 columns):
SortOrder                                        780 non-null int64
Region                                           780 non-null object
Region or State                                  780 non-null object
Year                                             780 non-null int64
Total land                                       780 non-null int64
Total cropland                                   780 non-null int64
Cropland used for crops                          780 non-null int64
Cropland used for pasture                        780 non-null int64
Cropland idled                                   780 non-null object
Grassland pasture and range                      780 non-null int64
Forest-use land (all)                            780 non-null int64
Forest-use land grazed                           780 non-null int64
Forest-use land not grazed                       780 non-null int64
All spec

In [5]:
# Sorting the index using the SortOrder column just so it looks cleaner. 
df.set_index('SortOrder')

Region Region or State  Year  Total land  Total cropland  \
SortOrder                                                                      
1          Northeast total       Northeast  1945      112402           25027   
2          Northeast total       Northeast  1949      112378           24557   
3          Northeast total       Northeast  1954      112378           22830   
4          Northeast total       Northeast  1959      112324           20973   
5          Northeast total       Northeast  1964      112223           19173   
6          Northeast total       Northeast  1969      112129           17853   
7          Northeast total       Northeast  1974      112129           17334   
8          Northeast total       Northeast  1978      112129           17501   
9          Northeast total       Northeast  1982      111730           16954   
10         Northeast total       Northeast  1987      111730           15707   
11         Northeast total       Northeast  1992      111401           14337   
12         Northeast total       Northeast  1997      111401           13402   
13         Northeast total       Northeast  2002      111390           13700   
14         Northeast total       Northeast  2007      111390           12967   
15         Northeast total       Northeast  2012      111123           12220   
16               Northeast           Maine  1945       19866            1490   
17               Northeast           Maine  1949       19866            1407   
18               Northeast           Maine  1954       19866            1274   
19               Northeast           Maine  1959       19848            1186   
20               Northeast           Maine  1964       19797            1002   
21               Northeast           Maine  1969       19789             735   
22               Northeast           Maine  1974       19789             653   
23               Northeast           Maine  1978       19789             689   
24               Northeast           Maine  1982       19837             629   
25               Northeast           Maine  1987       19837             577   
26               Northeast           Maine  1992       19753             591   
27               Northeast           Maine  1997       19753             466   
28               Northeast           Maine  2002       19752             478   
29               Northeast           Maine  2007       19752             462   
30               Northeast           Maine  2012       19740             390   
...                    ...             ...   ...         ...             ...   
856                Pacific          Oregon  1945       61664            4909   
857                Pacific          Oregon  1949       61642            5537   
858                Pacific          Oregon  1954       61642            5282   
859                Pacific          Oregon  1959       61599            5474   
860                Pacific          Oregon  1964       61574            5425   
861                Pacific          Oregon  1969       61557            5145   
862                Pacific          Oregon  1974       61557            5276   
863                Pacific          Oregon  1978       61557            5247   
864                Pacific          Oregon  1982       61558            5357   
865                Pacific          Oregon  1987       61558            5384   
866                Pacific          Oregon  1992       61441            5281   
867                Pacific          Oregon  1997       61441            5339   
868                Pacific          Oregon  2002       61438            5311   
869                Pacific          Oregon  2007       61438            4934   
870                Pacific          Oregon  2012       61432            4664   
871                Pacific      California  1945      100354           11363   
872                Pacific      California  1949      100314           13765   
873                P

## Question 1: Which region had the most "Grassland pasture and range" in 1974?

In [6]:
# I used a pivot table to create a dataframe with the values I want to sort through. 
df1 = df.pivot_table(values='Grassland pasture and range', index='Year', columns='Region')
df1

Region  Appalachian  Appalachian total  Corn Belt  Corn Belt total  \
Year                                                                 
1945        2486.75            13631.0     5265.2          26326.0   
1949        1854.75             9115.0     3312.6          16563.0   
1954        2097.50            10149.0     3366.0          16830.0   
1959        2528.25            12984.0     4361.2          21806.0   
1964        2186.50            10778.0     4067.0          20335.0   
1969        1639.00             8427.0     2789.6          13948.0   
1974        1371.25             7498.0     2738.6          13693.0   
1978        1115.75             5764.0     2402.0          12010.0   
1982        1163.50             6323.0     2632.0          13160.0   
1987        1226.25             6558.0     2568.6          12843.0   
1992        1150.75             5962.0     2465.0          12325.0   
1997         987.75             5442.0     2316.0          11579.0   
2002         987.25             6253.0     2615.8          13078.0   
2007        1759.00            10551.0     3277.8          16390.0   
2012        2287.00            13890.0     3639.8          18200.0   

Region  Delta States  Delta States total  Lake States  Lake States total  \
Year                                                                       
1945     2405.000000              7215.0  3355.333333            10066.0   
1949     2005.666667              6017.0  2050.333333             6151.0   
1954     2833.666667              8501.0  2095.666667             6287.0   
1959     3119.333333              9358.0  2755.333333             8266.0   
1964     3144.333333              9433.0  2828.333333             8485.0   
1969     2811.000000              8433.0  2058.333333             6175.0   
1974     2483.000000              7449.0  1763.333333             5290.0   
1978     1925.666667              5777.0  1571.666667             4715.0   
1982     2463.333333              7390.0  1859.666667             5579.0   
1987     2435.666667              7307.0  1823.666667             5471.0   
1992     2119.000000              6357.0  1759.333333             5278.0   
1997     1844.666667              5533.0  1664.666667             4995.0   
2002     2082.000000              6246.0  1798.000000             5393.0   
2007     2402.666667              7209.0  2495.000000             7486.0   
2012     3039.000000              9116.0  2804.666667             8414.0   

Region      Mountain  Mountain total   Northeast  Northeast total  \
Year                                                                
1945    41828.142857        339243.0  327.285714          10126.0   
1949    42786.571429        347861.0  216.571429           6928.0   
1954    40344.000000        330892.0  216.285714           7186.0   
1959    38063.142857        312832.0  215.142857           7999.0   
1964    38367.571429        314399.0  155.428571           7110.0   
1969    38223.857143        313478.0   94.714286           3162.0   
1974    37347.000000        307445.0   74.428571           3390.0   
1978    37281.571429        306508.0   63.142857           2963.0   
1982    36989.857143        304523.0   73.000000           2612.0   
1987    36731.000000        302263.0   61.285714           2777.0   
1992    36944.285714        303515.0   51.714286           2987.0   
1997    36826.142857        302658.0   52.142857           2827.0   
2002    36928.428571        302823.0   53.142857           3025.0   
2007    36963.428571        303397.0   82.000000           4627.0   
2012    40111.571429        326867.0  100.857143           4916.0   

Region  Northern Plains  Northern Plains total       Pacific  Pacific total  \
Year                                                                          
1945           20573.75                82295.0  18941.333333        56824.0   
1949           19263.75                77055.0  20183.333333        60550.0   
1954           19600.50   

In [7]:
# I selected the row of 1974 using the .loc method and sorted the values from max to min. 
df1.loc[1974].sort_values(ascending=False)

Region
Mountain total           307445.000000
Southern Plains total    112038.000000
Northern Plains total     73285.000000
Southern Plains           56019.000000
Pacific total             53761.000000
Mountain                  37347.000000
Northern Plains           18321.250000
Pacific                   17920.333333
Corn Belt total           13693.000000
Southeast total           11341.000000
Appalachian total          7498.000000
Delta States total         7449.000000
Lake States total          5290.000000
Northeast total            3390.000000
Southeast                  2835.250000
Corn Belt                  2738.600000
Delta States               2483.000000
Lake States                1763.333333
Appalachian                1371.250000
Northeast                    74.428571
Name: 1974, dtype: float64

### Ansewer 1: Region is Mountain (total) as it had 307445 grassland pasture and range in 1974.

## Question 2: How many states had at least 2,000 in the “Land in Urban areas” column for any year prior to 1987?


In [8]:
# I used a pivot table to create a dataframe with the values I want to sort through.
df2 = df.pivot_table(values='Land in urban areas', index='Region or State', columns='Year')
df2

Year             1945  1949  1954  1959  1964  1969  1974  1978  1982  1987  \
Region or State                                                               
Alabama           212   428   475   774   813   653   707  1408  1555  1834   
Appalachian      1096  1334  1508  2178  2317  2650  3133  4117  4685  5475   
Arizona            44    76    82   312   380   411   530   799   977  1194   
Arkansas          152   192   213   282   303   333   420   567   638   784   
California       1463  2166  1571  2456  2882  3191  3550  4007  4448  5224   
Colorado          106   131   151   230   256   336   430   546   663   792   
Connecticut       260   279   253   407   453   514   548   713   778   770   
Corn Belt        2651  2950  3253  4354  4534  5053  5462  6311  6729  7365   
Delaware           23    23    38    62    70    79    87   106   118   134   
Delta States      649   841   815  1118  1194  1200  1407  1849  2122  2416   
Florida           513   773   634  1003  1155  1442  1891  2397  2867  3643   
Georgia           362   440   495   765   831   822   928  1347  1571  1891   
Idaho              61    74    63    96    98   106   141   146   163   187   
Illinois          729   809   890  1142  1194  1385  1500  1671  1796  1905   
Indiana           339   398   416   583   608   790   906   979  1040  1191   
Iowa              413   456   445   530   530   456   477   655   677   746   
Kansas            186   227   242   299   305   370   435   478   536   636   
Lake States      1513  1784  1836  2653  2760  2629  2826  3567  3880  4081   
Louisiana         339   439   390   535   575   513   537   737   865   883   
Maine              97   104   113   173   176   201   214   559   605   612   
Massachusetts     456   452   479   752   781   897   992  1234  1340  1125   
Michigan          653   803   819  1133  1186  1248  1355  1482  1566  1632   
Minnesota         454   499   493   778   804   688   731  1216  1383  1466   
Mississippi       158   210   212   301   316   354   450   545   619   749   
Missouri          350   400   496   596   618   749   855   974  1067  1237   
Montana            49    59    58    74    77    75    90   111   127   142   
Mountain          634   867   786  1254  1414  1548  1957  2620  3138  3723   
Nebraska          140   152   137   163   168   173   201   223   243   269   
Nevada             27    47    41    53    78   125   184   215   262   300   
New Hampshire      61    66    67    90    97   151   163   350   367   412   
New Mexico        117   187   155   198   212   225   279   339   384   471   
North Carolina    341   360   410   596   631   683   832  1078  1255  1477   
North Dakota       44    55    55    66    68    62    78    90   102   112   
Northeast        3975  4126  4137  5821  6186  6632  7157  8993  9802  9945   
Northern Plains   435   514   512   615   630   685   807   893   991  1143   
Ohio              820   887  1006  1503  1584  1673  1724  2032  2149  2286   
Oklahoma          190   230   237   505   535   506   535  1060  1250  1471   
Oregon            146   197   184   273   291   323   382   432   471   519   
Pacific          1857  2722  2085  3218  3681  4117  4621  5215  5815  6755   
Rhode Island       97    91   102   126   130   151   154   170   173   190   
South Carolina    192   235   254   362   380   406   516   700   822  1005   
South Dakota       65    80    78    87    89    80    93   102   110   126   
Southeast        1279  1876  1858  2904  3179  3323  4042  5852  6815  8373   
Southern Plains   923  1269  1771  3006  3267  3003  3144  4801  5672  6632   
Tennessee         231   278   335   519   554   710   805  1197  1353  1650   
Texas             733  1039  1534  2501  2732  2497  2609  3741  4422  5161   
Utah              195   248   197   243   264   221   240   384   459   526   
Vermont            46    50    45    52    53    42    42    70    85    93   
Virginia          211   344   369   551   612   72

In [9]:
# I dropped columns and rows not needed to clean the data. 
df2.drop([1987,1992,1997,2002,2007,2012],axis=1,inplace=True)
df2.drop(['Appalachian','Corn Belt','Delta States','Lake States','Mountain','Northeast','Northern Plains','Pacific','Southeast','Southern Plains'],inplace=True)

In [10]:
# Using a conditional statement, I filtered the data to only show values equals or larger than 2000 and then dropped states with nan values.  
df2[df2>=2000.0].dropna(how='all')

Year             1945    1949  1954    1959    1964    1969    1974    1978  \
Region or State                                                               
California        NaN  2166.0   NaN  2456.0  2882.0  3191.0  3550.0  4007.0   
Florida           NaN     NaN   NaN     NaN     NaN     NaN     NaN  2397.0   
Ohio              NaN     NaN   NaN     NaN     NaN     NaN     NaN  2032.0   
Texas             NaN     NaN   NaN  2501.0  2732.0  2497.0  2609.0  3741.0   

Year               1982  
Region or State          
California       4448.0  
Florida          2867.0  
Ohio             2149.0  
Texas            4422.0

### Answer 2: California, Florida, Ohio,  and Texas.

## Question 3: What is the average value of the “Cropland used for pasture” column among all states within the Pacific and Mountain regions for 1964?


In [11]:
# I used a pivot table to create a dataframe with the values I want to sort through.
df3 = df.pivot_table(values='Cropland used for pasture', index='Year', columns='Region')
df3

Region  Appalachian  Appalachian total  Corn Belt  Corn Belt total  \
Year                                                                 
1945        1066.75             8305.0     2272.4          11362.0   
1949        1485.00            11205.0     2987.8          14939.0   
1954        1356.50            10306.0     2743.2          13716.0   
1959        1357.25             9498.0     2564.4          12822.0   
1964        1221.25             9457.0     2245.6          11228.0   
1969        1878.00            12428.0     3377.2          16886.0   
1974        1757.25            11516.0     3023.2          15116.0   
1978        1628.50            10648.0     2799.4          13997.0   
1982        1403.25             9066.0     2187.4          10937.0   
1987        1336.50             8748.0     2078.0          10390.0   
1992        1404.75             9073.0     2019.0          10095.0   
1997        1434.50             8943.0     1978.8           9894.0   
2002        1137.50             7151.0     1451.2           7255.0   
2007         555.25             3770.0      735.4           3677.0   
2012         176.25             1032.0      216.4           1082.0   

Region  Delta States  Delta States total  Lake States  Lake States total  \
Year                                                                       
1945     1177.333333              3532.0  1631.000000             4893.0   
1949     1960.000000              5880.0  1910.000000             5730.0   
1954     1963.666667              5891.0  1901.000000             5703.0   
1959     1977.333333              5932.0  1552.333333             4657.0   
1964     1624.333333              4873.0  1406.333333             4219.0   
1969     2228.000000              6684.0  1764.333333             5293.0   
1974     2124.666667              6374.0  1554.666667             4664.0   
1978     1861.000000              5583.0  1295.333333             3886.0   
1982     1470.666667              4412.0  1000.333333             3001.0   
1987     1337.000000              4011.0   934.333333             2803.0   
1992     1423.333333              4270.0   879.666667             2639.0   
1997     1362.333333              4087.0   833.333333             2500.0   
2002     1189.333333              3568.0   641.000000             1923.0   
2007      713.333333              2140.0   483.666667             1452.0   
2012      188.333333               566.0   142.000000              426.0   

Region    Mountain  Mountain total   Northeast  Northeast total  \
Year                                                              
1945    252.285714          1878.0   62.428571           2280.0   
1949    543.857143          4212.0  131.285714           4217.0   
1954    497.714286          3929.0  125.000000           3777.0   
1959    607.142857          4838.0   97.000000           3217.0   
1964    572.428571          4574.0   81.142857           2559.0   
1969    732.142857          5726.0   90.142857           3669.0   
1974    651.571429          5121.0   73.428571           3106.0   
1978    620.142857          4814.0   79.428571           3117.0   
1982    589.285714          4586.0   62.142857           2449.0   
1987    673.142857          5262.0   59.142857           2301.0   
1992    715.571429          5730.0   49.857143           2035.0   
1997    686.000000          5562.0   43.142857           1847.0   
2002    899.714286          7211.0   30.714286           1493.0   
2007    666.714286          5088.0   20.142857            887.0   
2012    296.000000          2329.0    6.857143            310.0   

Region  Northern Plains  Northern Plains total      Pacific  Pacific total  \
Year                                                                         
1945             795.00                 3180.0  1067.000000         3201.0   
1949            1168.00                 4672.0  1738.000000         5214.0   
1954             988.00                 3952.0  1482.000000     

In [12]:
# I selected both columns and took the mean value. 
df3[['Mountain total','Pacific total']].loc[1964].mean()

4169.0

### Answer 3: Average values for Mountain total and Pacific total in 1964 = 4169

## Question 4: Consider the original 13 colonies of England which became what is now the United States. Of the states that have land within the territory of those 13 colonies, which state made the largest contribution to its region's total “Forest-use land” in 2012?


In [14]:
# I used a pivot table to create a dataframe with the values I want to sort through.
df4 = df.pivot_table(values='Forest-use land (all)', index='Year', columns='Region or State')
df4

Region or State  Alabama  Appalachian  Arizona  Arkansas  California  \
Year                                                                   
1945               18748        63669    19466     19928       43891   
1949               18817        64476    19206     20020       44810   
1954               20766        68021    18412     19295       39398   
1959               20771        69311    18278     19891       39402   
1964               21749        73021    19798     20630       39406   
1969               21748        73132    17420     18237       39826   
1974               21333        73133    17420     18236       39826   
1978               21333        71876    17475     18230       37035   
1982               21179        70898    17846     17324       37043   
1987               21659        70469    17257     16896       36441   
1992               21941        71614    15968     17423       34679   
1997               21911        71495    16306     18392       32579   
2002               22922        71539    17608     18373       33780   
2007               22587        70819    16780     18596       26983   
2012               22800        70629    13000     18544       16991   

Region or State  Colorado  Connecticut  Corn Belt  Delaware  Delta States  \
Year                                                                        
1945                19533         1833      28954       442         51404   
1949                19103         1900      29455       442         52715   
1954                19523         1979      31033       454         51641   
1959                20913         1979      30270       392         53245   
1964                21351         1979      30407       391         54624   
1969                19387         2119      31115       390         50471   
1974                19387         1846      29128       390         50470   
1978                20009         1831      27789       390         49453   
1982                19809         1745      27841       394         47827   
1987                18837         1751      29119       388         47443   
1992                18837         1693      31270       386         48269   
1997                18781         1557      31322       376         50672   
2002                18925         1436      31304       376         50667   
2007                18236         1413      34264       383         52317   
2012                15138         1346      35437       331         52684   

Region or State    ...      Southeast  Southern Plains  Tennessee  Texas  \
Year               ...                                                     
1945               ...          72994            46318      11899  35737   
1949               ...          74926            47184      11911  36545   
1954               ...          78114            43099      12301  32810   
1959               ...          76855            37901      13758  27850   
1964               ...          78992            33071      13643  23892   
1969               ...          77061            32990      12820  24064   
1974               ...          76256            33339      12820  24043   
1978               ...          75078            31674      12820  23203   
1982               ...          73356            20890      12965  14080   
1987               ...          73500            20506      12863  13536   
1992               ...          73434            21670      13275  14548   
1997               ...          71938            18000      13265  11767   
2002               ...          73661            18007      13656  11774   
2007               ...          75150            24779      13913  17159   
2012               ...          76004            22013      13500  14357   

Region or State   Utah  Vermont  Virginia  Washington  West Virginia  \
Year                                                                   
1945              8462     3

In [15]:
# I selected the row of 2012 using the .loc method and sorted the values from max to min.
df4.loc[2012].sort_values(ascending=False)

Region or State
Mountain           98845
Southeast          76004
Appalachian        70629
Northeast          65670
Pacific            59188
Delta States       52684
Lake States        51918
Corn Belt          35437
Georgia            24352
Oregon             24116
Alabama            22800
Southern Plains    22013
Mississippi        19495
Michigan           19262
Arkansas           18544
Montana            18429
Washington         18081
North Carolina     18078
Maine              17191
California         16991
Idaho              16771
Wisconsin          16727
Minnesota          15929
Florida            15827
Virginia           15385
Colorado           15138
Missouri           15085
Louisiana          14645
Texas              14357
Tennessee          13500
New Mexico         13278
South Carolina     13025
Arizona            13000
West Virginia      11780
Utah                9810
Ohio                7864
Oklahoma            7656
Northern Plains     6178
Nevada              5417
Illinois 

### Answer 4 : Georgia had the largest contribution of its forest use land in 2012 with 24352.

## Question 5: Which region has had the largest shift in its land use between 1945 and 2012? Please explain your reasoning, including the data that you used to reach that conclusion.


In [16]:
# I used a pivot table to create a dataframe with the values I want to sort through.Then I dropped unwanted columns. 
df5 = df.pivot_table(values='All special uses of land', index='Year', columns='Region')
df55 = df5.drop(['Appalachian','Corn Belt','Delta States','Lake States','Mountain','Northeast','Northern Plains','Pacific','Southeast','Southern Plains'],axis=1)


In [17]:
df55

Region  Appalachian total  Corn Belt total  Delta States total  \
Year                                                             
1945               5767.0           7378.0              3085.0   
1949               6091.0           7557.0              3507.0   
1954               6092.0           7598.0              3556.0   
1959               6347.0           7664.0              3772.0   
1964               6510.0           7784.0              3874.0   
1969               5989.0           7222.0              3162.0   
1974               6005.0           7116.0              3171.0   
1978               6901.0           7637.0              3358.0   
1982               7094.0           7710.0              3470.0   
1987               7410.0           7338.0              3566.0   
1992               7622.0           7264.0              3710.0   
1997               7631.0           7444.0              3693.0   
2002               8891.0           8432.0              4419.0   
2007               8869.0           9411.0              4500.0   
2012               8849.0           8790.0              4569.0   

Region  Lake States total  Mountain total  Northeast total  \
Year                                                         
1945               5517.0         27443.0           6374.0   
1949               7400.0         23499.0           7797.0   
1954               7095.0         25352.0           7497.0   
1959               7792.0         28647.0           7703.0   
1964               7998.0         39747.0           8210.0   
1969               7621.0         40707.0           8525.0   
1974               7709.0         41821.0           8315.0   
1978               8328.0         45765.0           8680.0   
1982               8414.0         48302.0           9121.0   
1987              12203.0         48975.0           9827.0   
1992               9024.0         53937.0           9549.0   
1997               9049.0         50947.0           9836.0   
2002               9803.0         55137.0          10404.0   
2007              10130.0         62537.0          11214.0   
2012               9992.0         63420.0          11491.0   

Region  Northern Plains total  Pacific total  Southeast total  \
Year                                                            
1945                   7125.0        12117.0           5212.0   
1949                   7230.0        12474.0           5696.0   
1954                   7324.0        14745.0           6618.0   
1959                   7603.0        14621.0           7035.0   
1964                   8143.0        18948.0           7352.0   
1969                   6410.0        19521.0           7232.0   
1974                   6143.0        19663.0           7738.0   
1978                   6671.0        21106.0           8633.0   
1982                   6692.0        21890.0           8503.0   
1987                   6339.0        24974.0           8630.0   
1992                   6330.0        23284.0           9244.0   
1997                   6106.0        31228.0           8984.0   
2002                   7137.0        32343.0           9099.0   
2007                   8170.0        36821.0           9698.0   
2012                   7197.0        36648.0           9554.0   

Region  Southern Plains total  
Year                           
1945                   5001.0  
1949                   5799.0  
1954                   5760.0  
1959                   6098.0  
1964                   6740.0  
1969                   5880.0  
1974                   5728.0  
1978                   5926.0  
1982                   6083.0  
1987                   6011.0  
1992                   6425.0  
1997                   6840.0  
2002                   7110.0  
2007                   7951.0  
2012                   8169.0

In [18]:
# I used the .std method to get the standard deviation values for all the regions between 1945-2012. The standard devation values represents the shift between values from year to year. 
df55.std().sort_values(ascending=False)

Region
Mountain total           12952.634919
Pacific total             8224.101681
Lake States total         1583.797456
Northeast total           1426.000644
Southeast total           1389.590375
Appalachian total         1109.480973
Southern Plains total      859.869879
Northern Plains total      667.143234
Corn Belt total            638.470121
Delta States total         473.628074
dtype: float64

### Answer 5: Mountain total with standard deviaiton of 12952.634919.